java/junit.ui/src/org/netbeans/modules/junit/ui/actions/TestClassInfoTask.java

```diff
@@ -91,6 +92,23 @@ public void run(CompilationController controller) throws Exception {
                 }
             }
         }
+        if (methodName != null) {
```

Comment:
I agree. Are there any tests of similar code in the codebase I could use for reference? I think deep stubbing `CompilationController` in order to test this would make for a brittle test.

Reply:
@JaroslavTulach are there examples of similar code being tested in the codebase, without deep stubbing the `CompilationController`?